packages/wrangler/src/generate/index.ts

```diff
@@ -101,6 +104,27 @@ export async function generateHandler(args: GenerateArgs) {
 		throw new CommandLineArgsError(message);
 	}
 
+	if (isTemplateFolder(args.template)) {
```

Comment:
@irvinebroque as discussed, this change will delegate to c3 if a Cloudflare folder template is provided.

What I haven't done is pass through the template name itself, because there doesn't seem to be much overlap between the two, and if you provide a template that doesn't exist, it will error. We'll therefore have a confusing situation if it will occasionally succeed and occasionally not. For the sake of consistency, I have made it call c3 and accept all defaults (except `deploy` and `open`). Please advise if this is not the way we want to move forward here.